### PR TITLE
Switch sandbox clusters to 4.17

### PR DESCRIPTION
**Pull request by alexeykazakov**

### CODE REVIEW

File: ci-operator/config/codeready-toolchain/member-operator/codeready-toolchain-member-operator-master.yaml [hunk 1]
```diff
@@ -4,12 +4,12 @@ build_root:
 releases:
   initial:
     integration:
-      name: "4.15"
+      name: "4.17"
```
Comment (ranakan19) [inline ci-operator/config/codeready-toolchain/member-operator/codeready-toolchain-member-operator-master.yaml:7]: did we miss the upgrade to 4.16 here before?

**Review by ranakan19 — APPROVED** [review]

File: ci-operator/config/codeready-toolchain/member-operator/codeready-toolchain-member-operator-master.yaml (hunk 1 above)
Comment (alexeykazakov) [reply ci-operator/config/codeready-toolchain/member-operator/codeready-toolchain-member-operator-master.yaml:7]: Yep, we missed it when were switching to 4.16 last time.

**Review by mfrancisc — APPROVED** [review]
/lgtm

**Comment by alexeykazakov** [comment]
/pj-rehearse pull-ci-codeready-toolchain-host-operator-master-e2e

**Comment by alexeykazakov** [comment]
/pj-rehearse pull-ci-codeready-toolchain-registration-service-master-e2e

**Pull request merged**